cluster/node.go

```diff
@@ -78,7 +78,7 @@ func (n Node) RemoteURL() string {
 }
 
 func (n Node) IsReady() bool {
-	return n.State == NodeReady
+	return n.State == NodeReady && n.Priority <= maxPrio
```

Comment:
what is it about the priority system that is not good enough, that we need this? I don't see how this improves anything.
for the user the experience seems to get worse. instead of getting stale data from a node, they won't get data at all . e.g. instead of having a chart that has series with a gap, the series would be missing entirely.